codex-ui/dev/pages/components/Popup.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <PageHeader>
+    Popup with Confirm
+    <template #description>
+      A component that appears on top of other components
+    </template>
+  </PageHeader>
+  <Popup>
+    <template #trigger>
```

Comment:
lets put the Popup to the bottom of Playground.vue (and App.vue) and create a `usePopup()` service for showing the popup. Take a look the the Popover component for example

https://github.com/codex-team/notes.web/pull/250